simphony_ui/tests/test_openfoam_wrapper_creation.py

```diff
@@ -147,3 +166,38 @@ def test_raise(self):
         self.boundary_condition.type = 'coucou'
         with self.assertRaises(ValueError):
             get_boundary_condition_description(self.boundary_condition)
+
+
+class TestOpenfoamMeshCreation(unittest.TestCase):
+
+    def setUp(self):
+        self.openfoam_model = CustomOpenfoamModel()
+        self.openfoam_model.output_path = tempfile.mkdtemp()
+        self.openfoam_model.mesh_name = 'test_mesh'
+        self.openfoam_model.input_file = \
+            os.path.join(
+                os.path.dirname(os.path.dirname(__file__)),
+                'openfoam_input.txt'
+            )
+
+    def test_block_mesh_creation(self):
+        openfoam_wrapper = create_openfoam_wrapper(self.openfoam_model)
+        with cleanup_garbage(self.openfoam_model.output_path):
```

Comment:
using it like this will not perform cleanup. that routine is meant to be used in a different context, that is, in a setup to prevent failure **in the setup** to leave debris. if you use it like this, it will never cleanup anything, unless you get an exception during create_openfoam_mesh, which is not what you intend.